src/App.js

```diff
@@ -15,10 +30,32 @@ function App() {
           <Route path="/" element={<HomePage />}></Route>
           <Route path="search" element={<SearchPage />}></Route>
           <Route path="bookDetails" element={<BookDetailsPage />}></Route>
-          <Route path="exerciseDetails" element={<ExerciseDetailsPage />}></Route>
+          <Route
+            path="exerciseDetails"
+            element={<ExerciseDetailsPage />}
+          ></Route>
           <Route path="editExercise" element={<EditExercisePage />}></Route>
         </Routes>
       </SearchProvider>
+      <Routes>
```

Comment:
Why do these routes have to be separated from the existing routes that is already in place?